java/src/shared/model/game/Game.java

```diff
@@ -1521,13 +1534,23 @@ public JsonObject toJSON() {
      *
      * @param playerIndex Index of Player performing action
      */
-    public int rollNumber(int playerIndex) throws InvalidDiceRollException {
+    public int rollNumber(int playerIndex) throws InvalidDiceRollException, PlayerExistsException {
         assert playerIndex >= 0;
 
         Dice dice = new Dice(2);
         int roll = dice.roll();
```

Comment:
The RollNumberDTO includes the number that was rolled by the client. This will override that and roll a new number.